packages/web/src/components/common/notification-list/NotificationList.styles.ts

```diff
@@ -2,11 +2,18 @@ import { fonts } from "@constants/font.constant";
 import styled from "@emotion/styled";
 import { media } from "@styles/media";
 import mixins from "@styles/mixins";
+import { CSSProperties } from "react";
 
-export const NotificationListWrapper = styled.div`
+export interface NotificationProps {
+  width: CSSProperties["width"];
+}
+
+export const NotificationListWrapper = styled.div<NotificationProps>`
   position: absolute;
   top: 53px;
-  right: -150px;
+  right: ${({ width }) => {
+    return width && width > 1680 ? "-150px" : "0px";
```

Comment:
The type of `width` is either string or number. 
If it is a string, it cannot be compared to 1680.

Reply:
correct.